### PR TITLE
Add options to configure debug.

### PR DESCRIPTION
**Pull request by Natim**
- [x] Add tests.
- [x] Add a changelog entry.

r? @leplatrem

### CODE REVIEW

File: kinto/__main__.py [hunk 1]
```diff
@@ -31,10 +31,23 @@ def main(args=None):
...
+    # Defaults
+    parser.add_argument('-v', '--verbose', action='store_const',
+                        const=logging.INFO, dest='verbosity',
+                        help='Show all messages.')
+
+    parser.add_argument('-q', '--quiet', action='store_const',
+                        const=logging.CRITICAL, dest='verbosity',
+                        help='Show only critical errors.')
+
+    parser.add_argument('-D', '--debug', action='store_const',
```
Comment (leplatrem) [inline kinto/__main__.py]: Maybe a matter of taste but I think `--debug`` is enough. `-D` is not common right?

File: kinto/__main__.py [hunk 2]
```diff
@@ -31,10 +31,23 @@ def main(args=None):
                         required=False,
                         default=DEFAULT_CONFIG_FILE)
 
-    parser.add_argument('-v', '--version',
```
Comment (leplatrem) [inline kinto/__main__.py]: This deserves a note in the changelog

File: kinto/__main__.py [hunk 3]
```diff
@@ -82,13 +95,22 @@ def main(args=None):
                                    default=DEFAULT_PORT)
 
     # Parse command-line arguments
-    parsed_args = vars(parser.parse_args(args))
+    try:
+        parsed_args = vars(parser.parse_args(args))
+    except SystemExit as exc:
```
Comment (leplatrem) [inline kinto/__main__.py]: Can you add some comments ? What can happen here ?

File: kinto/__main__.py [hunk 4]
```diff
@@ -82,13 +95,22 @@ def main(args=None):
...
     config_file = parsed_args['ini_file']
     which_command = parsed_args['which']
 
     # Initialize logging from config.
-    logging.basicConfig(level=DEFAULT_LOG_LEVEL, format=DEFAULT_LOG_FORMAT)
+    kwargs = dict(format=DEFAULT_LOG_FORMAT)
+    if 'verbosity' in parsed_args and parsed_args['verbosity']:
+        kwargs['level'] = parsed_args['verbosity']
+
+    logging.basicConfig(**kwargs)
```
Comment (leplatrem) [inline kinto/__main__.py]: Why not:
```
level = parsed_args.get('verbosity', DEFAULT_LOG_LEVEL)
logging.basicConfig(level=level, format=DEFAULT_LOG_FORMAT)
```

File: kinto/tests/test_main.py [hunk 5]
```diff
@@ -132,3 +133,40 @@ def test_cli_start_with_reload_runs_pserve_with_reload(self):
...
+            mocked_logging.basicConfig.assert_called_with(
+                level=mocked_logging.DEBUG,
+                format=DEFAULT_LOG_FORMAT)
+
+    def test_cli_use_default_logging_logger(self):
+        with mock.patch('kinto.__main__.logging') as mocked_logging:
+            main(['--ini', TEMP_KINTO_INI, 'init',
+                  '--backend', 'memory'])
+            mocked_logging.basicConfig.assert_called_with(
+                format=DEFAULT_LOG_FORMAT)
```
Comment (leplatrem) [inline kinto/tests/test_main.py]: I don't think this test is necessary, `info` by default is fine no? logging default is warning btw

Comment (Natim) [inline kinto/tests/test_main.py]: In that case we should get rid of -v or change it for logging.DEBUG

File: kinto/__main__.py (hunk 4 above)
Comment (Natim) [inline kinto/__main__.py]: Because verbosity contains None

**Pull request merged**